pyro/contrib/funsor/infer/elbo.py

```diff
@@ -0,0 +1,52 @@
+# Copyright Contributors to the Pyro project.
+# SPDX-License-Identifier: Apache-2.0
+
+import weakref
+
+import pyro.ops.jit
+from pyro.infer import ELBO as OrigELBO
```

Comment:
nit: could you import this as `_ELBO` so it is not captured by `from pyro.infer.elbo import *`? Alternatively
```py
import pyro.infer

class ELBO(pyro.infer.ELBO):
    ...
```